### DISCUSSION TITLE
Simulation Window issue on Linux Mint

### DISCUSSION THREAD

**Discussion by Alb673** [meerk40t/meerk40t#2505] [Q&A] [answered]
Hi peeps, I have done a quick search, but haven't found something similar to my issue.

I recently downloaded the app image Linux v0.9.4020
I have noticed yesterday that while clicking simulation, it opens with a blank window. I have attached a video of the issue.
[Simulation -Issue-2024-06-11T210129+0100.webm](https://github.com/meerk40t/meerk40t/assets/23157880/07429a06-1f17-48be-91ab-b7274c983c34)

Cheers

**Comment by jpirnay** [comment, marked as answer]
Fixed since a couple of versions
